README.rst

```diff
@@ -1,9 +1,13 @@
-Repository for building and executing the `OCTOPUS code
-<http://octopus-code.org>`__ package in a Docker container. 
+Repository for running (and if desired building) the
+`OCTOPUS code <http://octopus-code.org>`__ package in a Docker container.
 
 Use case: run Octopus (for small calculations and tutorials) conveniently in a
 container, in particular on MacOS and Windows where compilation of Octopus may be non trivial.
 
+Octopus versions 13.0 and more recent are
```

Comment:
Why arent we advertising octopus 11.1 and above, we have their images uploaded as well.

Reply:
But not for the ARM64 architecture. I could try to create those for completeness. But let's do it outside this PR - this would take some time.